playbooks/roles/openconnect/templates/config.j2

```diff
@@ -33,6 +33,8 @@ ipv4-network = 192.168.1.0
 ipv4-netmask = 255.255.255.0
 ping-leases = false
 cisco-client-compat = true
+max-clients = {{ vpn_clients + 1 }}
+max-same-clients = {{ vpn_clients + 1 }}
```

Comment:
What is `max-same-clients`? Can you add a comment describing this var since it might not be as self-explanatory as `max-clients`?